.travis.yml

```diff
@@ -19,7 +19,7 @@ addons:
 env:
   global:
     - TRAVIS_NODE_VERSION="6"
-    - COMPOSER_ROOT_VERSION=1.3.x-dev
```

Comment:
Did you start this PR from `1` as a base? This should only belong in the `1` branch.